source/css/_schemes/Gemini/index.styl

```diff
@@ -139,51 +139,23 @@ use_seo = hexo-config('seo');
 }
 .post-body {
   h1 {
-    font-size: 1.6em;
```

Comment:
Screens: before and after?

Reply:
Before: https://github.com/theme-next/hexo-theme-next/issues/629
After: same as other schemes